uportal-war/src/main/webapp/WEB-INF/jsp/Search/searchLauncher.jsp

```diff
@@ -50,7 +50,7 @@
     </div>
 </div>
 
-<script type="text/javascript" src="<rs:resourceURL value="/rs/jquery/1.10.2/jquery-1.10.2.min.js"/>"></script>
+<script type="text/javascript" src="<rs:resourceURL value="/rs/jquery/1.11.0/jquery-1.11.0.min.js"/>"></script>
```

Comment:
I'm not confident the `searchLauncher` view should have it's own copy of jQuery.  I will look into it and take care of it as I process this PR.